components/permit-holders/modals/EditUserInformationModal.tsx

```diff
@@ -47,17 +57,53 @@ export default function EditUserInformationModal({ children }: EditUserInformati
 
   //   TODO: Add error states for each field (post-mvp)
 
-  const successfulEditToast = useToast();
+  const toast = useToast();
 
-  const handleSubmit = (event: SyntheticEvent) => {
-    event.preventDefault();
-    // TODO: Will be addressed in API hookup
-    onClose();
+  // Submit edited user information mutation
+  const [submitEditedUserInformation, { loading }] = useMutation<
+    UpdateApplicantResponse,
+    UpdateApplicantRequest
+  >(UPDATE_APPLICANT_MUTATION, {
+    onCompleted: data => {
+      if (data?.updateApplicant.ok) {
```

Comment:
Can `data` be undefined on a successful request?

Reply:
@j-bunk

Reply:
I'm not sure about the answer to that but we still hit this if statement line if the request is unsuccessful don't we? So if it can be undefined in the unsuccessful request then I think we should keep the `?`